device/rsvp/rsvpIngressP2PLsp.yaml

```diff
@@ -0,0 +1,83 @@
+components:
+  schemas:
+    Rsvp.LspInterface.P2PIngressLsp:
+      description: >-
+        Configuration for one RSVP Ingress point-to-point LSP.
+      type: object
+      required: [remote_address]
+      properties:
+        remote_address:
```

Comment:
what happens to this when we have IPv6 remote address or a v6 LSP in the future?  just trying to get a sense how we're thinking about extending this

Reply:
The plan is similar to Bgp we will have rsvp_ipv6_interfaces and rsvp_ipv6_lsp_endpoints to handle RSVP TE Ipv6 specification. In rsvp_ipv6_lsp_endpoints we will have p2p_ingress_lspsv6. The common properties if required we will then break up into objects which will not break otg, Note we believe no such requirements is present in mid to long term(based on existing enhancement received for rsvp te).

Reply:
Since we have changed IngressLsp to v4 specific keeping v6 open for introduction in future, marking this as resolved